app/src/main/java/com/teamwss/websoso/ui/memoPlain/MemoPlainActivity.kt

```diff
@@ -0,0 +1,121 @@
+package com.teamwss.websoso.ui.memoPlain
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.view.WindowManager
+import androidx.activity.viewModels
+import androidx.appcompat.app.AppCompatActivity
+import com.teamwss.websoso.databinding.ActivityMemoPlainBinding
+import com.teamwss.websoso.ui.memoWrite.MemoWriteActivity
+import kotlin.properties.Delegates
+
+class MemoPlainActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityMemoPlainBinding
+    private val memoPlainViewModel: MemoPlainViewModel by viewModels()
+    private var memoId by Delegates.notNull<Long>()
+
+    private lateinit var memoContent: String
+    private lateinit var userNovelTitle: String
+    private lateinit var userNovelAuthor: String
+    private lateinit var userNovelImage: String
+    private val dialogMemoDelete: DialogMemoDelete by lazy {
+        DialogMemoDelete(::finish)
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityMemoPlainBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        memoId = intent.getLongExtra("memoId", -1)
+        memoPlainViewModel.updateMemoId(memoId)
+        memoPlainViewModel.getMemo(memoId)
+
+        setTranslucentOnStatusBar()
+        setupLifecycleOwner()
+        setupDataBinding()
+        getUserNovelDataToIntent()
+        observeMemoId()
+        observeMemoContent()
+        onClickMemoPlainCancelButton()
+        onClickMemoDeleteButton()
+        onClickMemoEditButton()
+    }
+
+    private fun setTranslucentOnStatusBar() {
+        window.setFlags(
+            WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS,
+            WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS
+        )
+    }
+
+    private fun setupLifecycleOwner() {
+        binding.lifecycleOwner = this
+    }
+
+    private fun setupDataBinding() {
+        binding.memoPlainViewModel = memoPlainViewModel
+    }
+
+    private fun getUserNovelDataToIntent() {
+        userNovelImage = intent.getStringExtra("userNovelImage").toString()
+        userNovelTitle = intent.getStringExtra("userNovelTitle").toString()
+        userNovelAuthor = intent.getStringExtra("userNovelAuthor").toString()
+    }
+
+    private fun observeMemoId() {
+        memoPlainViewModel.memoId.observe(this) {
+            memoId = memoPlainViewModel.memoId.value!!
```

Comment:
!! << ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;; 엘비스연산자 써주세요 
님말대로 이거 절대절대절대 널일수가없다라고 생각한다면, 만약 널이 나오면 터뜨려도 합리적이라 생각함.

```suggestion
            memoId = memoPlainViewModel.memoId.value ?: throw IlegalArgumentException()
```

Reply:
에러 핸들링을 해야하는 상황인데, 진짜 말도안되는 상황이 나왔을 경우

앱을 터뜨린다 vs 터뜨리지 않는다.

예를들어서, 예전에 유튜브 숏츠 댓글이 모두 '0'으로 고정되었을때가 있었음. 이건 유튜브애들은 후자를 선택한 것.